R/api_client.R

```diff
@@ -88,9 +54,33 @@ ApiClient  <- R6::R6Class(
       }
     },
 
-    CallApi = function(url, method, query_params, header_params, body, ...) {
+    stop_for_status = function(resp) {
```

Comment:
why is this "stop"?

Reply:
`stop` is the way to throw an error in R, we do this if an API call fails (users could always tryCatch it if they want)